plugins/source/hackernews/client/spec.go

```diff
@@ -1,13 +1,16 @@
 package client
 
 import (
+	_ "embed"
 	"fmt"
 	"time"
 )
 
 type Spec struct {
-	ItemConcurrency int    `json:"item_concurrency"`
-	StartTime       string `json:"start_time"`
+	// The number of items to fetch concurrently
+	ItemConcurrency int `json:"item_concurrency" jsonschema:"minimum=1,default=100"`
+	// RFC3339 formatted timestamp. Syncing will begin with posts after this date. If empty everything is synced.
+	StartTime string `json:"start_time" jsonschema:"format=date-time"`
```

Comment:
you also need to check that `"start_time": ""` is allowed

Reply:
removed `automerge` bc of this

Reply:
Looks like the code gen doesn't support `oneof_type` with additional properties https://github.com/invopop/jsonschema/issues/113.
Regardless I believe the description is wrong, we should allow either a valid date-time or omit the property per  https://github.com/cloudquery/cloudquery/blob/5ecf6fc92eebfc0a17e129b7ba65c4e5d28265bb/plugins/source/hackernews/docs/overview.md?plain=1#L38

I updated the description to reflect this